### ISSUE TITLE
AttributeError: Can't get attribute 'RCNN' on <module 'network' 

### ISSUE THREAD

**Issue by kingstontan** [vloison/Handwritten_Text_Recognition#4]
I'm getting the following error while trying to make predictions. 
I fixed it temporarily by changing the class name in network.py from CRNN to RCNN. 
Should I be changing the class name? 

Also I managed to run the command after this change but my output in predictions.txt is empty.

Thanks a bunch

![image](https://user-images.githubusercontent.com/42428153/96156514-bcd17080-0f43-11eb-8bbf-d2308036b847.png)
from CRNN
![image](https://user-images.githubusercontent.com/42428153/96156966-4b45f200-0f44-11eb-9010-beb0100c78b6.png)
to RCNN
![image](https://user-images.githubusercontent.com/42428153/96157142-80524480-0f44-11eb-97ef-8a7ddb265241.png)
![image](https://user-images.githubusercontent.com/42428153/96157045-644ea300-0f44-11eb-8c8c-1f7cb218eb4e.png)

**Comment by melohux** [comment]
When we trained the model, we named the network as RCNN. Later we changed it to CRNN. That's where the problem comes from. Thanks a lot, we will fix it very soon. May I take a look at your test image ?

**Comment by kingstontan** [comment]
Got it. So I should leave it as RCNN for now? 

Here is the test image and here's the command I used. 

`python line_predictor.py --data_path images/ --model_path trained_networks/ICFHR2014_model_imgH64.pth --imgH 89 --save_model_path tesdir/`

![a01-000u-00](https://user-images.githubusercontent.com/42428153/96164007-328e0a00-0f4d-11eb-81ce-1c22938f0de2.png)

**Comment by melohux** [comment]
I have fixed these issues. Please update the code and now use the command line `python lines_predictor.py --data_path datapath  --model_path ./trained_networks/IAM_model_imgH64.pth --imgH 64` to test the image. Hope it will work well.

**Comment by kingstontan** [comment]
Hi Hux, I tried the new code with the following command, but the predictions were still empty. Using back the same test images.
`python line_predictor.py --data_path iam/ --model_path ./trained_networks/IAM_model_imgH64.pth --imgH 64`

The only 2 changes I made were 
1. the added `map_location=torch.device('cpu')` while loading model as I don't have an Nvidia gpu
2. I removed self.time in params.py 
![image](https://user-images.githubusercontent.com/42428153/96240192-2f3d6180-0fd3-11eb-87b9-0b01afbb6784.png)

Thank you so much for your time :)

![image](https://user-images.githubusercontent.com/42428153/96239690-99a1d200-0fd2-11eb-8bad-0414ae6bb2c5.png)
![image](https://user-images.githubusercontent.com/42428153/96239821-bfc77200-0fd2-11eb-84bc-5b13cb4891db.png)
